Tasks/CondaEnvironmentV0/task.json

```diff
@@ -0,0 +1,84 @@
+{
+    "id": "03DD16C3-43E0-4667-BA84-40515D27A410",
+    "name": "CondaEnvironment",
+    "friendlyName": "Conda Environment",
+    "description": "Create and activate a Conda environment.",
+    "helpMarkDown": "[More information](https://go.microsoft.com/fwlink/?linkid=873466)",
+    "category": "Package",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 136,
```

Comment:
update the minor version

Reply:
I believe the minor version currently on the service is 135 because this version never got picked up.  I can still change it if you want